.eslintignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+coverage/
+dist/
+bower_components/
+src/
```

Comment:
why would you ignore the whole src and test folder in eslintignore? won't this avoid the code getting linted?

Reply:
You are right in the point that it won't be linted. UI5Lab-Browser doesn't contain any src or test folder. We can delete it from the eslintignore